exchanges/bybit/bybit_wrapper.go

```diff
@@ -0,0 +1,1844 @@
+		}, nil
+
+	case asset.USDTMarginedFutures:
+		resp, err := by.GetActiveUSDTRealtimeOrders(ctx, pair, orderID, "")
+		if err != nil {
+			return order.Detail{}, err
+		}
+
+		if len(resp) != 1 {
+			return order.Detail{}, fmt.Errorf("%w, received %v orders", errExpectedOneOrder, len(resp))
+		}
+
+		return order.Detail{
+			Amount:         resp[0].Qty,
+			Exchange:       by.Name,
+			OrderID:        resp[0].OrderID,
+			ClientOrderID:  resp[0].OrderLinkID,
+			Side:           getSide(resp[0].Side),
+			Type:           getTradeType(resp[0].OrderType),
+			Pair:           pair,
+			Cost:           resp[0].CumulativeQty,
+			AssetType:      assetType,
+			Status:         getOrderStatus(resp[0].OrderStatus),
+			Price:          resp[0].Price,
+			ExecutedAmount: resp[0].Qty - resp[0].LeavesQty,
+			Date:           resp[0].CreatedAt,
+			LastUpdated:    resp[0].UpdatedAt,
+		}, nil
+
+	case asset.Futures:
+		resp, err := by.GetActiveRealtimeOrders(ctx, pair, orderID, "")
+		if err != nil {
+			return order.Detail{}, err
+		}
+
+		if len(resp) != 1 {
+			return order.Detail{}, fmt.Errorf("%w, received %v orders", errExpectedOneOrder, len(resp))
+		}
+
+		return order.Detail{
+			Amount:         resp[0].Qty,
+			Exchange:       by.Name,
+			OrderID:        resp[0].OrderID,
+			ClientOrderID:  resp[0].OrderLinkID,
+			Side:           getSide(resp[0].Side),
+			Type:           getTradeType(resp[0].OrderType),
+			Pair:           pair,
+			Cost:           resp[0].CumulativeQty,
+			AssetType:      assetType,
+			Status:         getOrderStatus(resp[0].OrderStatus),
+			Price:          resp[0].Price,
+			ExecutedAmount: resp[0].Qty - resp[0].LeavesQty,
+			Date:           resp[0].CreatedAt,
+			LastUpdated:    resp[0].UpdatedAt,
+		}, nil
+
+	case asset.USDCMarginedFutures:
+		resp, err := by.GetActiveUSDCOrder(ctx, pair, "PERPETUAL", orderID, "", "", "", "", 0)
+		if err != nil {
+			return order.Detail{}, err
+		}
+
+		if len(resp) != 1 {
+			return order.Detail{}, fmt.Errorf("%w, received %v orders", errExpectedOneOrder, len(resp))
+		}
+
+		return order.Detail{
+			Amount:         resp[0].Qty,
+			Exchange:       by.Name,
+			OrderID:        resp[0].ID,
+			ClientOrderID:  resp[0].OrderLinkID,
+			Side:           getSide(resp[0].Side),
+			Type:           getTradeType(resp[0].OrderType),
+			Pair:           pair,
+			Cost:           resp[0].TotalOrderValue,
+			AssetType:      assetType,
+			Status:         getOrderStatus(resp[0].OrderStatus),
+			Price:          resp[0].Price,
+			ExecutedAmount: resp[0].TotalFilledQty,
+			Date:           resp[0].CreatedAt.Time(),
+		}, nil
+
+	default:
+		return order.Detail{}, fmt.Errorf("%s %w", assetType, asset.ErrNotSupported)
+	}
+}
+
+// GetDepositAddress returns a deposit address for a specified currency
+func (by *Bybit) GetDepositAddress(ctx context.Context, cryptocurrency currency.Code, _, chain string) (*deposit.Address, error) {
+	return nil, common.ErrNotYetImplemented
```

Comment:
It looks like they now support it 😆 These are great methods to have, so please add support for these: https://bybit-exchange.github.io/docs/account_asset/#t-deposit_addr_info - They also support multichain transfers.